hack/test/e2e-qemu.sh

```diff
@@ -136,6 +136,15 @@ case "${WITH_SKIP_BOOT_PHASE_FINISHED_CHECK:-no}" in
       ;;
 esac
 
+case "${CUSTOM_CNI_NAME:-none}" in
```

Comment:
I would rather not bind together custom CNI and loadbalancer, let them be separate variables. 

```suggestion
case "${WITH_INCLUSTER_LOADBALANCER:-none}" in
```